### PR TITLE
Migrate remaining PHP scripts to NodeJS API

### PR DESCRIPTION
**Pull request by kevinhu**
- [x] Update server and frontend to use HTTPS (otherwise Facebook doesn't let us connect)
- [x] Move all queries with the MySQL database to Prisma (prisma.io)
- [x] Move all PHP scripts to Express API
- [x] Remove courses data from database.sql (since course info is all in Postgres now)
- [x] Refactor server code to pass Eslint
- [x] Fix async calls

Once this is merged, we can delete the `/web` and `/scripts` directories (didn't do it here since the diff would be gigantic).

### CODE REVIEW

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: api/prisma/schema.prisma [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+
+model StudentPetitions {
+  netId   String  @id @db.Char(8)
+  name    String  @db.VarChar(30)
+  comment String  @db.VarChar(4095)
+  public  Boolean
+  time    Int     @db.UnsignedInt
+}
+
+model Students {
```
[the remark this answers is not in the document]
Comment (kevinhu) [reply api/prisma/schema.prisma:60]: Agree, but not my call – this is autogenerated by Prisma from the MySQL database, and that's based off the already-existing table name. We should add this as a TODO when doing the migration though!

File: web/templates/EnableEvaluations.tpl [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+	<p><strong>CourseTable's evaluations data is sourced by Yale students, for Yale students.</strong> As a result, we need to verify that you have access to the evaluations before you get to see them.</p>
+
+	<p><a href="/Blog">Learn more about why this page exists here.</a></p>
+
+	<div class="row">
+		<div class="span6 offset3"><div class="well">
+			<h5>Answer these questions:</h5>
+			<div id="questionMessage" class="alert hide"></div>
+
+			<form action="/ProcessEnableEvaluations.php" method="post" id="questions">
```
[the remark this answers is not in the document]
Comment (kevinhu) [reply web/templates/EnableEvaluations.tpl:46]: Not sure why it's here either – I think there's some merge quirk with this branch, but I'll be deleting the whole /web directory after anyway.

File: web/templates/EnableEvaluations.tpl [hunk 3]
```diff
@@ -0,0 +1,103 @@
+{*
+	EnableEvaluations.tpl
+
+	Type: standalone template
+	Usage: YalePlus bluebook's separate page for uploading a data file
```
[the remark this answers is not in the document]
Comment (kevinhu) [reply web/templates/EnableEvaluations.tpl:5]: See below comment

**Pull request merged**